### PR TITLE
docs: versioning docs should show example versioned sidebar files

### PR DESCRIPTION
**Pull request by homotechsual**
## Motivation

Help clarify that versioned sidebars paths are relative to versioned content folders

## Test Plan

preview

### Test links

https://deploy-preview-9500--docusaurus-2.netlify.app/docs/versioning/

### CODE REVIEW

File: website/docs/guides/docs/versioning.mdx [hunk 1]
```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
 </Tabs>
 ```
 
+:::tip
+
+Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:
+
+```js
```
Comment (Josh-Cena) [inline website/docs/guides/docs/versioning.mdx:152]:
````suggestion
```json
````

...versioned sidebars are JSON, right? (You can see how rusty I am already.)

File: website/docs/guides/docs/versioning.mdx [hunk 2]
```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
...
+      "type": "autogenerated",
+      "dirName": "."
+    }
+  ]
+}
+```
+
+or for a manual sidebar:
+
+```js
```
Comment (Josh-Cena) [inline website/docs/guides/docs/versioning.mdx:165]: Same here?

````suggestion
```json
````

File: website/docs/guides/docs/versioning.mdx [hunk 3]
```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
...
+:::tip
+
+Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:
+
+```js
+{
+  "sidebar": [
+    {
+      "type": "autogenerated",
+      "dirName": "."
```
Comment (Josh-Cena) [inline website/docs/guides/docs/versioning.mdx:157]: You can have autogenerated versioned sidebars?! Does the versioning CLI produce this now?

File: website/docs/guides/docs/versioning.mdx [hunk 4]
```diff
@@ -145,6 +145,37 @@ versioned_sidebars/version-1.0.0-sidebars.json
 </Tabs>
 ```
 
+:::tip
+
+Versioned sidebar files are, like standard sidebar files plugin root relative - so for the example above your versioned sidebar file may look like:
```
Comment (Josh-Cena) [inline website/docs/guides/docs/versioning.mdx:150]:
```suggestion
Versioned sidebar files are, like standard sidebar files, relative to the content root for the given version — so for the example above, your versioned sidebar file may look like:
```

(Docs plugins use both `/docs` and `/versioned_docs` so there's not a single root to be "relative to".)

**Review by Josh-Cena — COMMENTED** [review]
I have a lot of questions. Maybe I'm far out of touch with the current versioning approach (I already was quite unfamiliar with versioning a year ago).

File: website/docs/guides/docs/versioning.mdx (hunk 3 above)
Comment (homotechsual) [reply website/docs/guides/docs/versioning.mdx:157]: I don't believe the CLI produces these but they do certainly appear to work.

Comment (slorber) [reply website/docs/guides/docs/versioning.mdx:157]: > Does the versioning CLI produce this now?

yes we don't substitute autogenerated items anymore for a while 👍 

CleanShot 2023-11-10 at 18 47 07@2x

**Pull request merged**